### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**

### CODE REVIEW

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100,) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
+Animal.alive = [];
```
Comment (DanilWeda) [inline src/herbivoresAndCarnivores.js:10]: alive needs to be static in Alive.
( maybe need to update deeps for test )

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
-  // write your code here
+  constructor(health, name) {
+    super(health, name);
+
+    Animal.alive.push(this);
+  }
+
+  bite(herbivore) {
+    if (herbivore.hidden === false && herbivore instanceof Herbivore) {
+      herbivore.health -= 50;
```
Comment (DanilWeda) [inline src/herbivoresAndCarnivores.js:34]: For 0, 50, and 100, the best way is to add in constant. ( for best readability)

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Good job! 
Please check minor comments for your solution! 
Thanks!

**Review by Manankin — APPROVED** [review]
Good job!

**Review by DanilWeda — APPROVED** [review]
Good job! 
Approved. 

For using static you need to add eslint-parser. ( for reference)

**Pull request closed without merging**